content/rs/concepts/compatibility.md

```diff
@@ -1,49 +1,40 @@
 ---
-Title: Redis Enterprise Software Compatibility with Open Source Redis
+Title: Redis Enterprise Software compatibility with open source Redis
 description:
 weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-Redis Enterprise Software (RS) is fully compatible with open source
+Redis Enterprise Software is compatible with open source
 Redis. Redis Labs contributes extensively to the open source Redis
-project and employs it inside of RS. As a rule, Redis Labs adheres to
+project and uses it inside of Redis Enterprise Software. As a rule, Redis Labs adheres to
 the open source's specifications and makes every effort to update
-RS with the latest version of Redis.
+Redis Enterprise Software with the latest version of open source Redis.
 
 ## Redis commands
 
-Any standard Redis client can be used with RS. That said, some of
-Redis functionality is not applicable in the context of RS, as
-follows:
+Any standard Redis client can be used with Redis Software. There is some
+Redis functionality (shown below) that's not applicable for Redis Software:
 
-- Shared databases are not supported in RS given their potential
-    negative impact on performance. Redis Labs recommends using
-    dedicated databases instead. Therefore the following commands are
-    blocked and produce an error when executed:
+- Shared databases aren't supported in Redis Software, because of the potential for
+    negative impact on performance. We recommend using
+    dedicated databases instead. The following commands are
+    blocked and produce an error when executed on a dedicated database:
     - MOVE
     - SELECT
-- Because data persistence and backups are managed from RS's
-    management UI, the following commands are blocked:
+- Data persistence, backups, and access controls are managed from the Redis Software
+    admin console. The following commands are blocked:
```

Comment:
Is this one thought or two?  Kind of reads like it's a new thought.  If so, it should be a separate bullet.